Sources/Rendering/Core/VolumeMapper/index.d.ts

```diff
@@ -278,6 +286,25 @@ export interface vtkVolumeMapper extends vtkAbstractMapper3D {
 	 */
 	setLAOKernelRadius(LAOKernelRadius: number): void;
 
+  /**
+   * Tells the mapper to only update the specified regions.
+   * 
+   * If there are zero regions, the mapper updates the entire volume texture.
+   * Otherwise, the mapper will only update the texture by the specified regions
+   * during the next render call.
+   * 
+   * This array is cleared after a successful render.
+   * @param regions 
+   */
+  setRegionsToUpdate(regions: VolumeRegion[]): boolean;
```

Comment:
I'm not sure what indent you are referring to here. It's all aligned formatting-wise.

Reply:
you have only 2 spaces for indent (not 3?).
Other functions in this files are indented with tabs.

Reply:
> Other functions in this files are indented with tabs.

Yeah, why is that? Shouldn't we be following vtk coding conventions and stick to 2 spaces instead of tabs?

Reply:
Here it is: https://github.com/Kitware/vtk-js/pull/3078